### ISSUE TITLE
Several warnings on clean install...

### ISSUE THREAD

**Issue by richb-hanover** [MattMcFarland/reactathon#28]
I just cloned the repo (d5c1c1b) to my OSX 10.10.5 box. The reactathon package built and ran as expected with `npm install && npm run dev`. However, there were several warnings about deprecated packages, and those that should be installed globally. 

This probably isn't a bug, but given your desire for a smooth installation experience, you may want to check  it out. Thanks for all this work!

The full output is at http://pastebin.com/8fXgwUrw, but the relevant portion is below:

```
bash-3.2$ nvm use 5.3.0
Now using node v5.3.0 (npm v3.3.12)
bash-3.2$ cd
bash-3.2$ cd github/reactathon/
bash-3.2$ npm install && npm run dev
npm WARN deprecated win-spawn@2.0.0: use [cross-spawn](https://github.com/IndigoUnited/node-cross-spawn) or [cross-spawn-async](https://github.com/IndigoUnited/node-cross-spawn-async) instead.
npm WARN deprecated lodash@1.0.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm
npm WARN deprecated has-color@0.1.7: Renamed to supports-color. If you're using chalk, upgrade to the latest version. https://github.com/chalk/supports-color
npm WARN prefer global marked@0.3.5 should be installed with -g
npm WARN prefer global js-beautify@1.5.10 should be installed with -g
npm WARN prefer global nodemon@1.8.1 should be installed with -g
npm WARN prefer global node-gyp@3.2.1 should be installed with -g
... etc...
```

**Comment by MattMcFarland** [comment]
@richb-hanover these warnings are mostly out of my control as they are coming from other dependencies that use them.  There's a possibility we could reduce the amount of warnings if we tracked down which packages are tied to a deprecated one, but we would still have to either replace them or see if their devs can update their package.

The preference for global warnings are also errors that should be ignored and occurr frequently within the nodejs ecosystem.  There is a movement right now that is shifting away from global, and a lot of people speaking out against using global installations because of security concerns so I purposely prefer items not be installed globally.

Another issue with the global preference is it would create an additional step for devs to have to install them globally themselves (with sudo, etc) before they could use reactathon, so my preference is to avoid globals.

TLDR: I dont think its really possible but we might be able to track down some of them and have developers of the modules update their packages.

**Comment by richb-hanover** [comment]
> TLDR: I dont think its really possible but we might be able to track down some of them and have developers of the modules update their packages.

Yes, I agree with your philosophy. I mentioned these messages because I follow the principle, "We promise not to fix it if we don't know it's broke..." :-)

I wonder if the README might contain a note that certain modules (named or not) may generate warnings, and to cite your preference for non-global installations. Thanks again for all this good work!

**Comment by MattMcFarland** [comment]
It's a good idea.. what do you think would be a good place in readme?  should we have another section called "notes", maybe in a "FAQ" section, or "troubleshooting" section??  What is your preference (if any)

**Comment by richb-hanover** [comment]
See my PR

**Comment by MattMcFarland** [comment]
Closing per the PR

**Comment by ToView** [comment]
I have also touched a similar problem. How do you fix it?

**Comment by richb-hanover** [comment]
@ToView  - I don't think you should use the Reactathon project as a basis for a new project. There haven't been any (important) updates in two years. All its libraries are outdated, and many techniques have improved.

I would look hard at create-react-app from Facebook as a starter project https://github.com/facebookincubator/create-react-app

**Comment by ToView** [comment]
Ok, thank you.

At 2017-07-10 20:24:29, "Rich Brown" <notifications@github.com> wrote:


@ToView - I don't think you should use the Reactathon project as a basis for a new project. There haven't been any (important) updates in two years. All its libraries are outdated, and many techniques have improved.

I would look hard at create-react-app from Facebook as a starter project https://github.com/facebookincubator/create-react-app
